pkg/kubelet/kubelet.go

```diff
@@ -1623,6 +1629,13 @@ func (kl *Kubelet) syncLoop(updates <-chan PodUpdate, handler SyncHandler) {
 				return
 			}
 			kl.podManager.UpdatePods(u, podSyncTypes)
+			if u.Op == cache.ADD {
+				// TODO: Break out SyncPods and check admission etc
+				for _, p := range u.Pods {
+					kl.podWorkers.UpdatePod(p, nil, func() {})
+				}
+				continue
+			}
 			unsyncedPod = true
 		case <-time.After(kl.resyncInterval):
```

Comment:
Would this change cause the existing pods to be sync'ed less often than intended? 
For example, what happens if we create one pod a second? Would existing pods get sync'ed at least every `resyncInterval`? This would affect pod deletions too.